pipelines.py

```diff
@@ -87,7 +87,7 @@ def _extract_answers(self, context):
             max_length=32,
         )
         
-        dec = [self.ans_tokenizer.decode(ids, skip_special_tokens=False) for ids in outs]
+        dec = [self.ans_tokenizer.decode(ids, skip_special_tokens=True) for ids in outs]
         answers = [item.split('<sep>') for item in dec]
```

Comment:
Sorry, I'm wondering if you set `skip_special_tokens=True`, will `item.split('<sep>')` still work? Will `<sep>` be skipped in decode?